systemd/usr/libexec/miriway-session-startup-snap

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -eu
+
+bindir=$(dirname "$0")
+if [ "${bindir}" != "" ]; then bindir="${bindir}/"; fi
+
+"${bindir}"miriway-session-startup "$@"
```

Comment:
```suggestion
source "$SNAP/usr/libexec/miriway-session-startup" "$@"
```

Reply:
Fixed! Using `source` or `exec` made it not work though